src/components/Table/components/RenderRowOrNestedRow/components/ExpandedButtonCell/ExpandedButtonCell.tsx

```diff
@@ -30,30 +29,27 @@ const ExpandedButtonCell: React.FC<Props> = ({
     <TableCell width={67} dataTestIdPrefix={dataTestIdPrefix} rowIndex={rowIndex} index={index}>
       <div>
         <div
-          data-testid="expanded-button"
```

Comment:
this will break QA tests we must let QA know

Reply:
I have informed @geokats7 about the change. I will also ping him later about the version of DS that this will get released.